.github/workflows/build-zip-file.yml

```diff
@@ -19,16 +44,26 @@ jobs:
         id: build_plugin
         uses: ./.github/actions/build
 
-      - name: "Unzip the file (prevents double zip issue)"
+      - name: "Add file size notice & delete build zip"
         run: |
-          echo ":information_source: Unzipping to a folder to prevent double-zip issue ([known limitation](https://github.com/actions/upload-artifact#zipped-artifact-downloads)). Increase in artifact size is expected since we're uploading the folder." >> $GITHUB_STEP_SUMMARY
-          unzip ${{ steps.build_plugin.outputs.release-filename }} -d zipfile
-      
+          echo ":information_source: Ignore the artifact size mentioned since GitHub calculates the size of the source folder instead of the zip file created." >> $GITHUB_STEP_SUMMARY
+          rm -rf release/${{ steps.build_plugin.outputs.release-filename }}
+
       - name: "Upload the zip file as an artifact"
         uses: actions/upload-artifact@v3
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           name: "woocommerce-payments"
-          path: zipfile
-          retention-days: 7
+          path: release
+          retention-days: 14
+
+  smoke-tests:
+    name: "Smoke tests"
+    needs: build-zip
+    if: ${{ inputs.skip-build-zip != true && inputs.skip-smoke-tests != true }}
```

Comment:
Can we use `! inputs.skip-smoke-tests` instead of `inputs.skip-smoke-tests != true`?
I remember I had some problems with booleans and if conditions back then, but I see you've been using both notations in the PR, so maybe it works nicely everywhere, and we can be consistent?

Reply:
Updated in https://github.com/Automattic/woocommerce-payments/pull/5586/commits/d15f4b18c70daa7a3431cb520699709a4e44a40e